acceptance/apps_test.go

```diff
@@ -1643,6 +1643,22 @@ userConfig:
 					),
 				)
 			})
+
+			It("scales up to a running workload", func() {
+				out, err := env.Epinio("", "app", "update", app, "-i", "3")
```

Comment:
Just a question. Not sure about what `-i` is here. Does this test check from 0 to 3?

Reply:
ok. nvm. It seems so. I  saw this later
```
BeforeEach(func() {
				app = catalog.NewAppName()
				By("make zero-instance app: " + app)
				env.MakeApp(app, 0, false)
```